tools/train_classifier.py

```diff
@@ -113,11 +132,11 @@
         backbone_network.cnn = backbone_network.cnn[0:-num_layers_to_finetune]
 
     # finetune the model
```

Comment:
```suggestion
    # Fine-tune the model
```

Reply:
I also prefer this spelling of "fine-tune", but so far we have written "finetune" everywhere else in the project :shrug: